decidim-admin-search-user/README.md

```diff
@@ -1,4 +1,4 @@
-# Decidim::Search::User
+# Decidim::Admin::Search::User
```

Comment:
@tramuntanal I renamed this module because it crash with the search engine of the Original Decidim.
Another possibility would be to move these files inside the customized module "Decidim::Admin::Extended", since it is an extension of the admin site. What do you think?

Reply:
Renamed!